autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -22,12 +25,34 @@ const CourseSelectColumn: React.FC = () => {
   );
   const term = useSelector<RootState, string>((state) => state.termData.term);
   const dispatch = useDispatch();
+  const [loading, setLoading] = React.useState(true);
+  const [wasCourseRemoved, setCourseRemoved] = React.useState(false);
+
+  const removeCallback = React.useCallback((id: number) => {
+    dispatch(removeCourseCard(id));
+    setCourseRemoved(true);
+  }, [dispatch, setCourseRemoved]);
+
+  const resetAnimations = React.useCallback(() => {
+    setCourseRemoved(false);
+  }, []);
```

Comment:
How come this doesn't have `setCourseRemoved` in it's deps, but `removeCallback` does? Does it make a difference either way?

Reply:
I guess I should add it in. It shouldn't make a difference, since I'm pretty sure the identity of the `setCourseRemoved` function stays the same from one render to the next.

Reply:
Honestly really the only reason I mentioned it was since I was confused how exhaustive deps didn't catch it, especially since it's literally just `useMemo` under the hood